lib/ca_testing/patches/capybara.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module CaTesting
+  module Patches
+    class Capybara < Base
+      private
+
+      def description
+        <<~DESCRIPTION
+          This patch is necessary when using the Safari browser. There is a bug in the text method and capybara
+          maintainers aren't willing to fix because according to them 'safari should fix their driver'.
```

Comment:
just thinking out loud, lets change this to something a bit milder like.

There is a bug in the `#text` method as Safari's w3c conformant endpoint isn't w3c conformant

Reply:
Please link to the bug - it's useful for future followup

Reply:
There isn't a direct bug report for this as I was told it wouldn't be actioned. But a good linked issue, which illustrates "more than" this issue (the w3c issue). Is this link: https://github.com/teamcapybara/capybara/issues/2213

Reply:
That would at least give someone a head start in the future.